scss/components/list-group.scss

```diff
@@ -0,0 +1,81 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-list-group
+  .tn-list-group__item
+    .tn-list-group__button
+    .tn-list-group__addon
+      .tn-list-group__control
+      .tn-list-group__label
+
+*/
+$block: ns(list-group);
+
+$tn-list-group--border-color: lighten(tn-color(neutral, 2), 5%);
+$tn-list-group-hover: tn-color(action);
+$tn-list-group--input-width: tn-space(7) !default;
+
+.#{$block} {
+  @include reset;
+  vertical-align: top;
+  min-height: $tn-forms-height--input-text;
+
+  &__item{
+    border: 1px solid $tn-list-group--border-color;
```

Comment:
1px is used four times. it's worth to put this in a var ($tn-list-group--border-width).